app/assets/javascript/pageflow/linkmap_page/page_type.js

```diff
@@ -187,14 +187,21 @@ pageflow.pageType.register('linkmap_page', _.extend({
     wrapper
       .attr('data-width', template.data('videoWidth'))
       .attr('data-height', template.data('videoHeight'));
-
-    var videoPlayer = this.videoPlayer = new pageflow.VideoPlayer.Lazy(template, {
+    
+    var options = {
       volumeFading: true,
       fallbackToMutedAutoplay: true,
-
       width: '100%',
       height: '100%'
-    });
+    }
+    if (pageflow.browser.has('mobile platform')) {
+      // to enable autoplay videos on mobile, 'muted' and 'playsinline' needs to be set
+      // video can be unmuted on user interaction
+      options.autoplay = true;
```

Comment:
The `autoplay` option causes the video element to start playing as soon as it is created. Since the video element is created on page prepare (see `prepare` method below), which in turn is called when the user enters an adjacent page, the muted video already starts playing when the user is not yet on the page. Since `play` is called anyway when the page is activated, the `autoplay` option can be removed.